client/tso_dispatcher.go

```diff
@@ -330,51 +335,51 @@ func (c *tsoClient) handleDispatcher(
 		tbc.clear()
 		c.wg.Done()
 	}()
-	// Call updateTSOConnectionCtxs once to init the connectionCtxs first.
-	c.updateTSOConnectionCtxs(dispatcherCtx, dc, &connectionCtxs)
-	// Only the Global TSO needs to watch the updateTSOConnectionCtxsCh to sense the
-	// change of the cluster when TSO Follower Proxy is enabled.
-	// TODO: support TSO Follower Proxy for the Local TSO.
-	if dc == globalDCLocation {
```

Comment:
Why remove this check?

Reply:
Previously, this goroutine is only used to check the connection updating when the TSO Follower Proxy is enabled for the global DC location. This PR makes it a permanent daemon to update the connection regularly.